### PR TITLE
resource_control: enhance observability

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/15935
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statement-summary-tables.md [hunk 1]
```diff
@@ -402,6 +403,15 @@ Transaction-related fields:
 - `AVG_AFFECTED_ROWS`: The average number of rows affected.
 - `PREV_SAMPLE_TEXT`: When the current SQL statement is `COMMIT`, `PREV_SAMPLE_TEXT` is the previous statement to `COMMIT`. In this case, SQL statements are grouped by the digest and `prev_sample_text`. This means that `COMMIT` statements with different `prev_sample_text` are grouped to different rows. When the current SQL statement is not `COMMIT`, the `PREV_SAMPLE_TEXT` field is an empty string.
 
+Fields related to Resource Control:
+
+- `Max_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
```
Comment (glorv) [inline statement-summary-tables.md:408]:
```suggestion
- `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
- `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
```

Comment (glorv) [reply statement-summary-tables.md:408]: missing a field, also added in docs-cn

**Review by glorv — COMMENTED** [review]
LGTM

**Review by songrijie — APPROVED** [review]
LGTM

File: statement-summary-tables.md [hunk 2]
```diff
@@ -402,6 +403,16 @@ Transaction-related fields:
...
 
+Fields related to Resource Control:
+
+- `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
+- `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
+- `AVG_REQUEST_UNIT_WRITE`: the average number of write RUs consumed by SQL statements.
+- `MAX_REQUEST_UNIT_WRITE`: the maximum number of write RUs consumed by SQL statements.
+- `AVG_REQUEST_UNIT_READ`: the average number of read RUs consumed by SQL statements.
+- `MAX_REQUEST_UNIT_READ`: the maximum number of read RUs consumed by SQL statements.
+- `RESOURCE_GROUP`: the resource group bound to SQL statements.
```
Comment (hfxsd) [inline statement-summary-tables.md:414]:
```suggestion
- `AVG_REQUEST_UNIT_WRITE`: the average number of write RUs consumed by SQL statements.
- `MAX_REQUEST_UNIT_WRITE`: the maximum number of write RUs consumed by SQL statements.
- `AVG_REQUEST_UNIT_READ`: the average number of read RUs consumed by SQL statements.
- `MAX_REQUEST_UNIT_READ`: the maximum number of read RUs consumed by SQL statements.
- `AVG_QUEUED_RC_TIME`: the average waiting time for available RU when executing SQL statements.
- `MAX_QUEUED_RC_TIME`: the maximum waiting time for available RU when executing SQL statements.
- `RESOURCE_GROUP`: the resource group bound to SQL statements.
```

File: system-variables.md [hunk 3]
```diff
@@ -3251,6 +3251,7 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
     - `start_ts`: The start timestamp of the transaction.
     - `for_update_ts`: The `for_update_ts` of the previously executed DML statement. This is an internal term of TiDB used for tests. Usually, you can ignore this information.
     - `error`: The error message, if any.
+    - `last_ru_consumption`: Consumed [RU](/tidb-resource-control.md#what-is-request-unit-ru) for executing the DML statement.
```
Comment (glorv) [inline system-variables.md:3254]:
```suggestion
    - `ru_consumption`: Consumed [RU](/tidb-resource-control.md#what-is-request-unit-ru) for executing the DML statement.
```
remove the misleading `last_` prefix

File: statement-summary-tables.md [hunk 4]
```diff
@@ -86,7 +86,8 @@ The following is a sample output of querying `statements_summary`:
 
 > **Note:**
 >
-> In TiDB, the time unit of fields in statement summary tables is nanosecond (ns), whereas in MySQL the time unit is picosecond (ps).
+> - In TiDB, the time unit of fields in statement summary tables is nanosecond (ns), whereas in MySQL the time unit is picosecond (ps).
+> - Starting from v7.6.0, for clusters with [resource control](/tidb-resource-control.md) enabled, `statements_summary` will be aggregated by resource group, i.e., the same statements executed in different resource groups will be collected as different records.
```
Comment (ran-huang) [inline statement-summary-tables.md:90]:
```suggestion
> - Starting from v7.6.0, for clusters with [resource control](/tidb-resource-control.md) enabled, `statements_summary` will be aggregated by resource group, for example, the same statements executed in different resource groups will be collected as different records.
```

File: system-variables.md [hunk 5]
```diff
@@ -3251,6 +3251,7 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
     - `start_ts`: The start timestamp of the transaction.
     - `for_update_ts`: The `for_update_ts` of the previously executed DML statement. This is an internal term of TiDB used for tests. Usually, you can ignore this information.
     - `error`: The error message, if any.
+    - `ru_consumption`: Consumed [RU](/tidb-resource-control.md#what-is-request-unit-ru) for executing the DML statement.
```
Comment (ran-huang) [inline system-variables.md:3254]:
```suggestion
    - `ru_consumption`: Consumed [RU](/tidb-resource-control.md#what-is-request-unit-ru) for executing the statement.
```
中文没有 DML？

**Review by ran-huang — APPROVED** [review]

**Review by glorv — APPROVED** [review]

**Pull request merged**